script.js

```diff
@@ -6,21 +6,21 @@ window.onload = () => {
     // console.log('');
```

Comment:
[OPTIONAL]
KEEP YOUR PROJECT PROFESSIONAL AND CLEAN.
- REMOVE COMMENTED LINES.